docs/operator-notes.md

```diff
@@ -51,6 +51,12 @@ If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting fi
 
 If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot.
 
+Ignition allows to erase a block device without creating a filesystem on it. In this case, its behavior is controlled by the `wipeFilesystem` and  the `format` flags in the `filesystem` section.
+
+If `wipeFilesystem` is set to true and `format` is set to `none`, Ignition will always wipe any preexisting filesystem and avoid creating a new filesystem on the block device.
+
+If `wipeFilesystem` is set to false and `format` is set to `none`, Ignition will fail if a filesystem is detected on the block device.
```

Comment:
Hmm, maybe it would read better if it were part of each of the above paragraphs? E.g.

> If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting filesystem and create the desired filesystem (or skip creation if the format is set to `none`). Note this will result in any data on the old filesystem being lost.

and

> If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot. Similarly, if the format is set to `none`, then any preexisting filesystem will cause Ignition to fail.